api/handlers/users.js

```diff
@@ -0,0 +1,55 @@
+const Pool = require('pg').Pool;
+const pool = new Pool({
+    user: 'masteruser',
+    password: "ecommdbpw",
+    host: 'ecommdb.covnxnivrrc5.ap-southeast-2.rds.amazonaws.com',
+    database: "ecommdb",
+    port: 5432,
+    ssl: true
+})
+module.exports.getAllUsers = async (event) => {
```

Comment:
This `getAllUsers` API should be secured so that only authorised administrators can access it. Right now anyone could use it anonymously to download all user data, which is dangerous because that's other people's personal information. 

The same applies to `getUserById` below, except that as well as administrators it would be OK for other users to call that as long as it's looking up their own user ID... you'd have to check with Amazon Cognito that the user ID matches.